### PR TITLE
feat: allow admins to prevent the icon in the top left corner from sp…

### PR DESCRIPTION
**Pull request by dadlerj**
…inning on hovers

fixes https://github.com/sourcegraph/sourcegraph/issues/4574

logo-spin

You can't see it in my gif, but I'm hovering over the icon both BEFORE I update the config and AFTER. It spins before, doesn't after.

### CODE REVIEW

**Comment by dadlerj** [comment]
Note I added this to 3.5 in the changelog assuming it gets in, but @slimsag you should make the call whether this makes it in or not.

File: web/src/globals.d.ts [hunk 1]
```diff
@@ -120,6 +120,9 @@ interface SourcegraphContext {
         light?: BrandAssets
         /** Override style for dark themes */
         dark?: BrandAssets
+
+        /** Prevent the icon in the top-left corner of the screen from spinning */
```
Comment (slimsag) [inline web/src/globals.d.ts]:
```suggestion
        /** Prevents the icon in the top-left corner of the screen from spinning. */
```

**Comment by dadlerj** [comment]
Should I wait to merge until the code nav team can review, or do you want to get this in as-is today? @slimsag you can make the merge call if needed sooner!

**Comment by slimsag** [comment]
I'm going to do a subsequent patch release for this.

**Comment by slimsag** [comment]
but it's fine as-is so merging. It just won't be in 3.5.

File: web/src/nav/GlobalNavbar.scss [hunk 2]
```diff
@@ -22,6 +22,9 @@
         height: 1.5rem;
 
         width: 1.5rem;
+    }
+
+    &__logo-animated {
```
[the remark this answers is not in the document]
Comment (dadlerj) [reply web/src/nav/GlobalNavbar.scss]: Thanks! I _thought_ it was supposed to be like that, but I also saw https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/web/src/nav/GlobalNavbar.scss#L13:13 a couple lines above it and figured that they were, as far as I could tell, both somewhere between modifiers and "nouns" (or distinct objects, whatever the terminology is in bem). What's the conceptual difference here, or was the one above also incorrect?

[the remark this answers is not in the document]
Comment (felixfbecker) [reply web/src/nav/GlobalNavbar.scss]: `__logo-link` and `logo` are different elements - `__logo-link` is on the `<a>` (or `<div>` parent), `logo` is on the ``. `animated` is also on `__logo-link`, so it is a _modifier_ of `__logo-link` and should actually be `__logo-link--animated`.

So the difference is that they _not_ distinct objects and animated is not a noun but an adjective.

**Pull request merged**